pkg/ccl/testccl/workload/schemachange/schemachange_test: TestWorkloadMultiRegion failed

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkloadMultiRegion [failed](https://mesolite.cluster.engflow.com/invocations/default/4d85ce4b-b96a-4b1a-8027-527df103d60b?testReportRun=9&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [a41e86dbb2bd18447a4ccf69d6c5f887827cd7d7](https://github.com/cockroachdb/cockroach/commits/a41e86dbb2bd18447a4ccf69d6c5f887827cd7d7):


```
        	            				pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:50
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: invalid input value for enum enum_w0_143: "qo" (SQLSTATE 22P02)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3544
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:612
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:700
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.runSchemaChangeWorkload.runSchemaChangeWorkload.func5.func7
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:172
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:93
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1771
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: invalid input value for enum enum_w0_143: "qo" (SQLSTATE 22P02)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkloadMultiRegion
{
 "message": "Inspect Job Errors",
 "jobs": [
  {
   "jobId": 1168194636311986177,
   "status": "no errors reported"
  },
  {
   "jobId": 1168194537330704385,
   "status": "no errors reported"
  }
 ]
}
Schema Workload Stats
Total Schema Statements Executed = 129
Total Schema Statements Succeeded = 129
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 79
Total Transactions Rolled Back = 60
Total Transactions Executed = 139
    schema_change_external_test.go:136: backup, tracing data, and system table dumps in /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/shard_2_of_2_run_9_of_10/test.outputs/logTestWorkloadMultiRegion312238881
    schema_change_external_test.go:145: running DROP with use_declarative_schema_changer = off
    panic.go:694: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkloadMultiRegion312238881
--- FAIL: TestWorkloadMultiRegion (71.60s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=9</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkloadMultiRegion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-63062

pkg/ccl/testccl/workload/schemachange/schemachange_test.TestWorkloadMultiRegion [failed](https://mesolite.cluster.engflow.com/invocations/default/c2cffbce-f3d1-4a6f-b0eb-64683354463a?testReportRun=4&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvd29ya2xvYWQvc2NoZW1hY2hhbmdlOnNjaGVtYWNoYW5nZV90ZXN0) on master @ [a41e86dbb2bd18447a4ccf69d6c5f887827cd7d7](https://github.com/cockroachdb/cockroach/commits/a41e86dbb2bd18447a4ccf69d6c5f887827cd7d7):


```
 }
}
    schema_change_external_test.go:183: 
        	Error Trace:	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:183
        	            				pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:50
        	Error:      	Received unexpected error:
        	            	***UNEXPECTED ERROR; Received an unexpected execution error.: ERROR: invalid input value for enum enum_w0_46: "huul" (SQLSTATE 22P02)
        	            	(1) forced error mark
        	            	  | "fatal error when running txn"
        	            	  | github.com/cockroachdb/errors/withstack/*withstack.withStack::
        	            	Wraps: (2)
        	            	Wraps: (3) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*opStmt).executeStmt
        	            	  | 	pkg/workload/schemachange/operation_generator.go:3544
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).runInTxn
        	            	  | 	pkg/workload/schemachange/schemachange.go:612
        	            	  | github.com/cockroachdb/cockroach/pkg/workload/schemachange.(*schemaChangeWorker).run
        	            	  | 	pkg/workload/schemachange/schemachange.go:700
        	            	  | pkg/ccl/testccl/workload/schemachange/schemachange_test_test.runSchemaChangeWorkload.runSchemaChangeWorkload.func5.func7
        	            	  | 	pkg/ccl/testccl/workload/schemachange/schema_change_external_test.go:172
        	            	  | golang.org/x/sync/errgroup.(*Group).Go.func1
        	            	  | 	external/org_golang_x_sync/errgroup/errgroup.go:93
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1771
        	            	Wraps: (4) ***UNEXPECTED ERROR; Received an unexpected execution error.
        	            	Wraps: (5) ERROR: invalid input value for enum enum_w0_46: "huul" (SQLSTATE 22P02)
        	            	Error types: (1) *markers.withMark (2) *schemachange.ErrorState (3) *withstack.withStack (4) *errutil.withPrefix (5) *pgconn.PgError
        	Test:       	TestWorkloadMultiRegion
{
 "message": "Inspect Job Errors",
 "jobs": [
  {
   "jobId": 1168468948234829825,
   "status": "no errors reported"
  }
 ]
}
Schema Workload Stats
Total Schema Statements Executed = 84
Total Schema Statements Succeeded = 84
Total Schema Statement Expected Failures = 0
Total Transactions Committed = 44
Total Transactions Rolled Back = 35
Total Transactions Executed = 79
    schema_change_external_test.go:136: backup, tracing data, and system table dumps in /mnt/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/testlogs/pkg/ccl/testccl/workload/schemachange/schemachange_test/shard_2_of_2_run_4_of_10/test.outputs/logTestWorkloadMultiRegion97264030
    schema_change_external_test.go:145: running DROP with use_declarative_schema_changer = off
    panic.go:694: -- test log scope end --
test logs left over in: outputs.zip/logTestWorkloadMultiRegion97264030
--- FAIL: TestWorkloadMultiRegion (55.01s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=4</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestWorkloadMultiRegion.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/workload/schemachange/operation_generator.go
+++ b/pkg/workload/schemachange/operation_generator.go
@@ -5832,6 +5832,7 @@ func (og *operationGenerator) createTrigger(ctx context.Context, tx pgx.Tx) (*op
 		{code: pgcode.UndefinedColumn, condition: true},
 		{code: pgcode.UndefinedObject, condition: true},
 		{code: pgcode.InvalidParameterValue, condition: true},
+		{code: pgcode.InvalidTextRepresentation, condition: true},
 	})
 
 	return opStmt, nil

